### DISCUSSION TITLE
Question about querying custom metagenomic proteins agains ESM metagenomic Atlas database.

### DISCUSSION THREAD

**Discussion by szimmerman92** [facebookresearch/esm#494] [ESM Metagenomic Atlas]
Hello,

Thank you for the amazing resource. I have a metagenomic database of over 100 million proteins from many different host and environmental microbiomes. I would like to query my protein sequences against the ESM metagenomic Atlas. Is there a way to download the MMseq2 database to do this? Or do you have another way to query against the ESM metagenomic Atlas? I would use the API provided, but I fear that 100 million proteins would be too many requests for the API. 

The database I have can be seen [here](https://www.biorxiv.org/content/10.1101/2022.07.19.500741v1) in this manuscript on bio archive if you are interested.  Thank you very much.

Best,
Sam Zimmerman

**Comment by tomsercu** [comment]
We didn't provide the mmseqs2 database but it should be trivial to create it from the fasta?
https://github.com/facebookresearch/esm/tree/main/scripts/atlas#full-database
> We also provide the fasta file for the full dataset: s3://dl.fbaipublicfiles.com/esmatlas/v0/full/mgnify90.fasta

https://dl.fbaipublicfiles.com/esmatlas/v0/full/mgnify90.fasta

Let me know if that will work.

And you're right, you won't be able to query 100 million sequences with our API. But mmseqs will handle it well
